### PR TITLE
Bug fix, add before and after in all comp page

### PR DESCRIPTION
**Pull request by liweijian**
This PR is going to:

1. Fix a bug in **src/screenshotbot/magick.lisp** (An unexpected `q` there)
2. Add `"half-page"` config in **src/screenshotbot/dashboard/image.lisp**
3. Add **before** and **after** in All Pixel Comparisons page, which would looks like

triple-row

I have tested it:)

### CODE REVIEW

**Comment by tdrhq** [comment]
I'm going to quickly test this, but otherwise the code looks great

File: src/screenshotbot/magick.lisp [hunk 1]
```diff
@@ -28,7 +28,7 @@
   (restart-case
       (call-with-semaphore
        *semaphore*
-       (lambda ()q
```
[the remark this answers is not in the document]
Comment (tdrhq) [reply src/screenshotbot/magick.lisp:31]: I recently reinstalled my Linux server that runs Jenkins, and I didn't configure it to send me notifications that my build was broken :/

**Comment by tdrhq** [comment]
Let me patch the 'q' while we review this. But it looks like the CSS might need a bit of work.(url)

image

The comparison view currently only works on the full image. So you have two half-page images, and a full image, which results in that rendering. For now, if you just keep each image as is, I'll go back and make them (including the comparison, this was on my todo list) into a half-page.

**Comment by tdrhq** [comment]
The 'q' patch: https://github.com/screenshotbot/screenshotbot-oss/commit/3bf2b58f1be52f08d73fbd88f7fef3de9092c24c

File: src/screenshotbot/compare.lisp [hunk 2]
```diff
@@ -310,19 +320,10 @@
                 collect
                 <div class= "image-comparison-wrapper" >
                 <h3>,(screenshot-name before)</h3>
```
Comment (tdrhq) [inline src/screenshotbot/compare.lisp:322]: comment for L318. In the (prepare-image-comparison ...) just add a `:size :half-page` as a parameter. That should fix the issue I mentioned.
(I added code to support image sizes for comparisons in https://github.com/screenshotbot/screenshotbot-oss/commit/931b8eda972fb399cbea24f1456454be69578657 and https://github.com/screenshotbot/screenshotbot-oss/commit/ad7f991c2e67446a7eb704a3445a94ef8bc7db84)

Comment (liweijian) [reply src/screenshotbot/compare.lisp:382]: Or may I change the other 2 images to `full-page` ?

**Comment by tdrhq** [comment]
Looks good, I'm going to merge it in. (I'll be merging it manually, not through GitHub)

**Comment by tdrhq** [comment]
Merged https://github.com/screenshotbot/screenshotbot-oss/commit/64b7eb8dbd7a5c0cf41aeae2ccfa9139f36e24f5 and https://github.com/screenshotbot/screenshotbot-oss/commit/0b6c8e0b0848a7559410fab69c5945ef4ad6113f

**Pull request closed without merging**